origin-discovery/src/apollo/db.js

```diff
@@ -7,7 +7,7 @@ const db = require('../models')
  * @param listingIds
  * @return {Promise<Array>}
  */
-async function getListings (listingIds) {
+async function getListings (listingIds, hiddenIds, featuredIds) {
```

Comment:
nit: document the new params hiddenIds and featuredIds
Also maybe add a TODO saying we should read those directly from DB within this method as opposed to getting them as params.

Reply:
Good point 👍